website/versioned_docs/version-0.4.x/set-up-your-environment.mdx

```diff
@@ -484,14 +501,142 @@ IMPORTANT: Ceramic API running on 0.0.0.0:7007
 
 By this point you should have your development environment and all configurations in place to get started working on your application.
 
+## Frequently Asked Questions
+
+---
+
+Some questions and issues come up more often than others. We've compiled a list of the most common ones here.
+
+<details>
+  <summary>Which setup method is better: wheel or JavaScript package managers?</summary>
+  <div>
+    <div>
+      Wheel is the recommended and the easiest way to configure your working environment and install
+      all the necessary dependencies. We highly recommended going with wheel if you are just
+      starting out with Ceramic. Everything will be taken care of for you.
+    </div>
+  </div>
+</details>
+<details>
+  <summary>Which operating systems are supported?</summary>
+  <div>
+    <div>
+      It's best to run Ceramic and ComposeDB on Linux or a Mac. You can also run it on Windows but
+      you'll have to use WSL2 (Windows Subsystem for Linux). See the supported operating systems
+      section at the top.
+    </div>
+  </div>
+</details>
+
+<details>
+  <summary>Which node version is preferred?</summary>
```

Comment:
This is kind of obvious, but let's explicitly say `Node.js` instead of `node` where possible. Most of devs understand we are talking about `Node.js` but the ambiguity sometimes comes from the fact that we have Ceramic nodes too so it can get confusing.